acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuValidatorNode.java

```diff
@@ -16,28 +16,41 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Path;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.function.Consumer;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.testcontainers.containers.Network;
 import org.testcontainers.utility.MountableFile;
+import tech.pegasys.teku.bls.BLSPublicKey;
 import tech.pegasys.teku.infrastructure.unsigned.UInt64;
+import tech.pegasys.teku.provider.JsonProvider;
 import tech.pegasys.teku.test.acceptance.dsl.tools.deposits.ValidatorKeystores;
 
 public class TekuValidatorNode extends Node {
   private static final Logger LOG = LogManager.getLogger();
+  private static final int VALIDATOR_API_PORT = 9052;
 
+  private final SimpleHttpClient httpClient;
+  private final JsonProvider jsonProvider = new JsonProvider();
   private final TekuValidatorNode.Config config;
   private boolean started = false;
   private Set<File> configFiles;
 
   private TekuValidatorNode(
       final Network network, final DockerVersion version, final TekuValidatorNode.Config config) {
     super(network, TEKU_DOCKER_IMAGE_NAME, version, LOG);
+    this.httpClient = new SimpleHttpClient();
     this.config = config;
+    if (config.configMap.containsKey("Xvalidator-api-enabled")) {
+      container.withExposedPorts(VALIDATOR_API_PORT);
```

Comment:
We could always expose this port - it won't hurt if nothing is listening inside the container.

Reply:
well, i thougth that, but then all the validator tests failed...